### PR TITLE
[1LP][RFR] Fixed test Service Start for Azure and EC2

### PR DESCRIPTION
**Pull request by niyazRedhat**
Fix test_service_start for azure. ec2 

{{pytest: --long-running --use-provider ec2west  --use-provider azure cfme/tests/ssui/test_ssui_myservice.py -k test_service_start}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0d2fd30a78b36e347e313bf23dff1de13980adf9:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 59:28: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 61:28: **E123** *closing bracket does not match indentation of opening bracket's line*
- :bangbang: Line 75:50: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 18f1c0e51b191378982a21e2ac891a128ceb5a99:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 59:28: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 1]
```diff
@@ -66,17 +68,16 @@ def test_service_start(appliance, setup_provider, context,
...
         my_service = MyService(appliance, catalog_item.name)
-        if provider.one_of(InfraProvider):
+        if provider.one_of(InfraProvider, EC2Provider, AzureProvider):
             # For Infra providers vm is provisioned.Hence Stop option is shown
+            # For Azure, EC2 Providers Instance is provisioned.Hence Stop option is shown
             my_service.service_power(power='Stop')
             view = my_service.create_view(DetailsMyServiceView)
-            view.notification.assert_message(
-                "{} was {}.".format(catalog_item.name, 'stopped'))
+            assert view.power_operations.item_enabled('Start')
```
Comment (rlbabyuk) [inline cfme/tests/ssui/test_ssui_myservice.py]: Question: Can be this button enabled when the process of stopping is not finished?

Comment (niyazRedhat) [reply cfme/tests/ssui/test_ssui_myservice.py]: @rlbabyuk  yes, start button gets enabled while the stopping process is not yet finished.

Comment (bsquizz) [reply cfme/tests/ssui/test_ssui_myservice.py]: There's a TODO in the `service_power` method to verify the VM states change using the REST API. That will be the ideal way to verify that the service was actually stopped. But I am curious, why wasn't `view.notification.assert_message` working?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2488160809796a0325e076e2e43578bbecfe2f85:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 59:14: **E131** *continuation line unaligned for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 86a26802a805ffce1d9ecd3fc7516d6ad79eaacc:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 60:18: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 839dcf96ba9a1ba1c9e5ac9610fa490a93dc3675:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 60:28: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 62:28: **E123** *closing bracket does not match indentation of opening bracket's line*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by bsquizz** [comment]
Hey @niyazRedhat where are we at with this one? Just checking to see if I need to review this again

**Comment by niyazRedhat** [comment]
@bsquizz Hey I will update this PR with latest changes very soon.

File: cfme/tests/ssui/test_ssui_myservice.py (hunk 1 above)
Comment (niyazRedhat) [reply cfme/tests/ssui/test_ssui_myservice.py]: I'll write test using API later. I have written it using UI itself. it was long pending on me.

**Pull request merged**